configs/brvs/spring-context.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version = "1.0" encoding = "UTF-8"?>
+
```

Comment:
Why xml configuration?
Why not java?

Reply:
To be able to reconfigure a set of rules (or other components) without recompiling